server/constants/db.js

```diff
@@ -4,11 +4,32 @@ exports.EXPERIMENT_METRICS_TABLE_NAME = "experiment_metrics";
 exports.EXPOSURES_TABLE_NAME = "exposures";
 exports.METRICS_TABLE_NAME = "metrics";
 exports.CONNECTION_TABLE_NAME = "connection";
-exports.GET_EXPT_METRICS_QUERY = `SELECT * FROM experiments e
-                                 JOIN experiment_metrics em
-                                 ON e.id=em.experiment_id
-                                 WHERE flag_id = $1
-                                 ORDER BY id DESC;`;
+exports.KEYS_TABLE_NAME = "keys";
+exports.GET_METRIC_DATA = `
+  SELECT em.experiment_id,
+    em.metric_id,
+    m.name,
+    m.type,
+    em.mean_test,
+    em.mean_control,
+    em.interval_start,
+    em.interval_end,
+    em.p_value
+  FROM experiment_metrics em
+  JOIN metrics m
+    ON em.metric_id = m.id
+  WHERE em.experiment_id = $1
+`;
+exports.GET_EXPT_METRICS_QUERY = `SELECT e.* ,
+                                    em.metric_id,
+                                    m.name,
+                                    m.type,
+                                    em.mean_test,
+                                    em.mean_control,
+                                    em.interval_start,
+                                    em.interval_end,
+                                    em.p_value
+                                FROM experiments e
```

Comment:
There's half a query here, and it's going to cause errors bc of mismatched quotes. Were you trying to delete the whole thing?

Reply:
Just resolved this. Was trying to merge stuff.